snakemake/workflow.py

```
@@ -1804,6 +1804,15 @@ def check_may_use_software_deployment(method):
 
                 rule.conda_env = ruleinfo.conda_env
 
+            invalid_rule = not (
```

Ruff (0.8.2)

Local variable `invalid_rule` is assigned to but never used

Remove assignment to unused variable `invalid_rule`

(F841)